packages/bpk-component-slider/src/form-test.tsx

```diff
@@ -67,9 +67,12 @@ describe('BpkSlider form-test for single-thumb slider', () => {
   });
 
   it('should emit change event when both sides of slider value are changed for two-thumb slider', async () => {
-    const handleChange = jest.fn();
+    const formValidation = jest.fn();
     const Wrap = () => {
-      const [sliderValue, setSliderValue] = useState<number[]>([0, 100]);  // two-thumb slider with min and max value
+      const [sliderValue, setSliderValue] = useState<number[]>([0, 100]); // two-thumb slider with min and max value
+      useEffect(() => {
+        document.addEventListener('change', formValidation);
+      }, []);
```

Comment:
How come this is within the `Wrap` component? Is there a reason for that?

Reply:
It's to setup the eventListener on render so that formValidation can be mocked and valdiated against.

Reply:
Why does it need to be setup on render?